.github/workflows/release.yml

```
@@ -56,8 +101,63 @@ jobs:
           git commit -a -m "Update changelog"
           git push "https://${{ env.CI_COMMIT_AUTHOR }}:${{ secrets.TF_DEVEX_COMMIT_GITHUB_TOKEN }}@github.com/${{ github.repository }}.git"
 
+  cdktfDocs:
+    needs:
+      - releasedVersion
+      - buildBinary
+      - changelog # So the commit is pushed for the changelog
+    runs-on: ubuntu-latest
+    container:
+      image: docker.mirror.hashicorp.services/hashicorp/jsii-terraform
+    env:
+      CHECKPOINT_DISABLE: "1"
+    timeout-minutes: 120
+    steps:
+      - uses: actions/checkout@v3.1.0
```

Per SEC-047 all external GitHub Actions usage should be pinned by SHA

```suggestion
      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
```